src/carousel.tsx

```diff
@@ -827,16 +1171,44 @@ export const Slideshow = ({
                     })}
                     {...getPrefixedDataAttributes(dataAttributes, 'SlideShow')}
                 >
-                    <ThemeVariant isInverse={false}>
-                        <BaseTouchable
-                            className={styles.slideshowPrevArrowButton}
-                            aria-label={texts.carouselPrevButton || t(tokens.carouselPrevButton)}
-                            onPress={goPrev}
-                            disabled={!showPrevArrow}
-                        >
-                            <IconChevronLeftRegular />
-                        </BaseTouchable>
-                    </ThemeVariant>
+                    {items.length > 1 &&
+                        (withControls ? (
+                            <ThemeVariant variant="media">
+                                <Inline space="between" className={styles.slideshowControlsContainer}>
+                                    {!!autoplay && (
+                                        <div className={styles.slideshowAutoplayControlContainer}>
+                                            <CarouselAutoplayControl
+                                                isAutoplayEnabled={isAutoplayEnabled}
+                                                isAtLastPage={currentIndex === items.length - 1}
+                                                onAutoplayChanged={(autoplayEnabled: boolean) => {
+                                                    if (
+                                                        currentIndex === items.length - 1 &&
+                                                        autoplayEnabled
+                                                    ) {
+                                                        goToPage(0);
+                                                    }
+                                                    setShouldAutoPlay(autoplayEnabled);
+                                                }}
+                                            />
+                                        </div>
+                                    )}
+                                    {bulletsContainer}
+                                    <CarouselPageControls
```

Comment:
When withBullets is false, prev/next controls move left, they should stay right:

https://mistica-fv8042n18-flows-projects-65bb050e.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8BlANgS1gZwBYQHcAdAOwAIyCMAXXAIQFc00ZrsBeYAMwEM1sYAX1IUaMALYdgAbREUySAJLieAcxhlsAJzDsiIXNWoAHbPAD054xjDYG4gHTH81CNnMBGAEwAGH598ffTIebGMYMGoAJR5qDAg9EA8ANngATmDzAD4AGjkKJRV1TR1EwxMzS2tbeycXNwC-RqCQELCI6Nj4xJT0zNy5AF1hEmzSEEEgA

Reply:
fixed ✏️